Sources/TripKitUI/views/cell accessory/ButtonAccessoryView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ButtonAccessoryView.swift
+//  TripKitUI-iOS
+//
+//  Created by Jules Gilos on 7/27/22.
+//  Copyright © 2022 SkedGo Pty Ltd. All rights reserved.
+//
+
+import TripKit
+
+import RxSwift
+import RxCocoa
+
+class ButtonAccessoryView: UIView {
```

Comment:
As discussed separately, this isn't needed in this repo, and can be just in the TripGo repo.